app/src/main/java/com/esaudev/wizeline/data/remote/api/BitsoApi.kt

```diff
@@ -0,0 +1,25 @@
+package com.esaudev.wizeline.data.remote.api
+
+import com.esaudev.wizeline.data.remote.responses.AvailableBooksResponse
+import com.esaudev.wizeline.data.remote.responses.OrderBookResponse
+import com.esaudev.wizeline.data.remote.responses.TickerResponse
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface BitsoApi {
+
+    @GET("available_books/")
```

Comment:
It's not necessary to put "/" after the name of the service  :D 
```suggestion
    @GET("available_books")
```